src/transaction-runner.coffee

```diff
@@ -274,7 +272,7 @@ class TransactionRunner
     if typeof(hook) == 'string'
       @runSandboxedHookFromString hook, data, callback
 
-  configureTransaction: (transaction, callback) =>
+  configureTransaction: (transaction) =>
```

Comment:
This is unrelated change, see https://github.com/apiaryio/dredd/pull/906/files#r152850808